Fluid/Values/FluidValue.cs

```diff
@@ -10,16 +10,13 @@ public abstract class FluidValue : IEquatable<FluidValue>
 #pragma warning restore CA1067
     {
         [Obsolete("WriteTo is obsolete, prefer the WriteToAsync method.")]
-        public abstract void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo);
-
-        public virtual ValueTask WriteToAsync(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo)
+        public virtual void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo)
         {
-#pragma warning disable CS0618 // Type or member is obsolete
-            WriteTo(writer, encoder, cultureInfo);
-#pragma warning restore CS0618 // Type or member is obsolete
-            return default;
+            WriteToAsync(writer, encoder, cultureInfo).GetAwaiter().GetResult();
```

Comment:
This is breaking everything. Current implementations of `WriteTo` that ignore the obsolete message or are already built have the code to run. That's what while [obsolete] is here the async one calls `WriteTo`. It's just if you implement `WriteToAsync` that it's picked up.

So it's fine to have `virtual void WriteTo` such that we don't HAVE to implement it, but that's the only change to make.

Reply:
That's work too